src/apps/ubeswap/positions.e2e.ts

```diff
@@ -7,8 +7,18 @@ describe('getPositionDefinitions', () => {
       NetworkId['celo-mainnet'],
       '0x2b8441ef13333ffa955c9ea5ab5b3692da95260d',
     )
-    // Simple check to make sure we got some definitions
-    expect(positions.length).toBeGreaterThan(0)
+    // Simple checks to make sure we got some definitions
+
+    // ube v2 pool definitions
+    expect(
+      positions.filter((p) => p.type === 'app-token-definition').length,
+    ).toBeGreaterThan(0)
+
+    // ube v2 farm definitions and ube v3 pools are both contract definitions. We could
+    // call displayProps to distinguish v2 farms from v3 pools for better coverage.
```

Comment:
Good idea! Do you mind doing it?

Reply:
Done, it looks a bit messy tbh I think, I think we could use the actual runtime for these types of tests in the future since we're reimplementing a lot of the logic that is already there (except mocking the actual data) to be able to call `displayProps`.

Reply:
Oh right yes would be better with with the runtime.